### PR TITLE
feat(pSTAKE): Add new app pSTAKE with token fetcher

### PR DESCRIPTION
**Pull request by vedantd**
## Description

* Adds new app integration pSTAKE to zapper
*  Includes a token fetcher for stkBNB Token

### CODE REVIEW

**Review by immasandwich — COMMENTED** [review]
Replace the `assets/pSTAKElogo.png` file with `logo.png` like
https://s2.coinmarketcap.com/static/img/coins/200x200/15996.png

**Review by immasandwich — COMMENTED** [review]
There's no need for changes in pnpm-lock.yaml, so revert those changes 🙏

File: src/apps/pstake/binance-smart-chain/pstake.Stake.token-fetcher.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    @Inject(PstakeContractFactory) private readonly pstakeContractFactory: PstakeContractFactory,
+  ) { }
+
+  async getPositions() {
+    const address = '0xc2e9d07f66a89c44062459a47a0d2dc038e4fb16';
+    const multicall = this.appToolkit.getMulticall(network);
+    const contract = this.pstakeContractFactory.stakedBnbToken({ address, network });
+
+    const baseTokenDependencies = await this.appToolkit.getBaseTokenPrices(network);
+    await new Promise(r => setTimeout(r, 2000));
```
Comment (immasandwich) [inline src/apps/pstake/binance-smart-chain/pstake.Stake.token-fetcher.ts:33]: Remove this promise wait

**Comment by immasandwich** [comment]
Few comments, otherwise looks great!

File: src/apps/pstake/binance-smart-chain/pstake.Stake.token-fetcher.ts (hunk 1 above)
Comment (vedantd) [reply src/apps/pstake/binance-smart-chain/pstake.Stake.token-fetcher.ts]: Done

**Comment by vedantd** [comment]
Can you please review. Thanks.

File: src/apps/pstake/binance-smart-chain/pstake.Stake.token-fetcher.ts (hunk 1 above)
Comment (vedantd) [reply src/apps/pstake/binance-smart-chain/pstake.Stake.token-fetcher.ts]: @immasandwich any more changes?

Comment (vedantd) [reply src/apps/pstake/binance-smart-chain/pstake.Stake.token-fetcher.ts]: > Replace the `assets/pSTAKElogo.png` file with `logo.png` like https://s2.coinmarketcap.com/static/img/coins/200x200/15996.png

Done

Comment (vedantd) [reply src/apps/pstake/binance-smart-chain/pstake.Stake.token-fetcher.ts]: @immasandwich @kafitz @maximebedard @mikeshultz

**Comment by vedantd** [comment]
@immasandwich @kafitz @maximebedard @mikeshultz Can this be reviewed?

**Comment by vedantd** [comment]
@immasandwich @kafitz @maximebedard @mikeshultz Can you please re review?

**Comment by vedantd** [comment]
HEY @immasandwich  can. you review soon pls?

**Review by immasandwich — APPROVED** [review]

**Comment by SamIam-0x** [comment]
Hello @vedantd  - thanks for submitting this! 

qq- do you have an ENS or address you'd want us to attribute this contribution to? We are building out a leaderboard to give credit to users who contributed to Zapper, with potential bonus points of value in the future.

Let me know, thanks!
Sam

**Pull request merged**